external-applications/quarkus-workshop-super-heroes/src/test/java/io/quarkus/ts/openshift/heroes/workshop/HeroesOpenShiftIT.java

```diff
@@ -0,0 +1,175 @@
+package io.quarkus.ts.openshift.heroes.workshop;
+
+import io.quarkus.ts.openshift.common.AdditionalResources;
+import io.quarkus.ts.openshift.common.ManualDeployment;
+import io.quarkus.ts.openshift.common.OpenShiftTest;
+import io.quarkus.ts.openshift.common.injection.TestResource;
+import org.hamcrest.core.Is;
+import org.junit.jupiter.api.MethodOrderer;
+import org.junit.jupiter.api.Order;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.TestMethodOrder;
+
+import java.net.URL;
+
+import static io.restassured.RestAssured.given;
+import static io.restassured.RestAssured.when;
+import static javax.ws.rs.core.HttpHeaders.ACCEPT;
+import static javax.ws.rs.core.HttpHeaders.CONTENT_TYPE;
+import static javax.ws.rs.core.MediaType.APPLICATION_JSON;
+import static javax.ws.rs.core.Response.Status.*;
+import static org.hamcrest.Matchers.is;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+@OpenShiftTest
+@ManualDeployment(appName = "quarkus-workshop-hero", httpRoot = "/", knownEndpoint = "/")
+@AdditionalResources("classpath:openjdk-11-rhel7.yaml")
+@AdditionalResources("classpath:heroes-database.yaml")
+@AdditionalResources("classpath:hero.yaml")
+@TestMethodOrder(MethodOrderer.OrderAnnotation.class)
```

Comment:
You're not using the `@Order` annotation :-)

Reply:
Actually you are, but not on all tests. Weird, but if it works for you, then sure :-)

Reply:
I think it's ok, test without annotation are independent

those 3 annotated need to go in create - update - delete order